### ISSUE TITLE
StatusFile missing from docs

### ISSUE THREAD

**Issue by tommoor** [nodegit/nodegit.github.com#40]
It's mentioned several times however there doesn't appear to be a definition for this object. Eg returned from `repo.getStatus`:

http://www.nodegit.org/api/repository/#getStatus

**Comment by johnhaley81** [comment]
You're right! https://github.com/nodegit/nodegit/blob/master/lib/status_file.js needs to be decorated similar to how https://github.com/nodegit/nodegit/blob/master/lib/convenient_hunks.js is broken out and decorated.
